src/App.js

```diff
@@ -1,16 +1,25 @@
 import {Routes, Route} from 'react-router-dom';
 
-import Home from './pages';
-import Details from './pages/details.js';
+import HomePage from './pages';
+import DetailPage from './pages/details.js';
+import PlannerPage from './pages/planner.js';
 import {GlobalStyle} from './styles';
 
 export default function App() {
 	return (
 		<>
 			<GlobalStyle />
 			<Routes>
-				<Route path="/" element={<Home />} />
-				<Route path="/:idFromUrl" element={<Details />} />
+				<Route
+					path="/"
+					element={
+						<>
+							<HomePage />
+							<PlannerPage />
```

Comment:
Why do you add two pages here and not add the `PlannerPage` component directly to the `HomePage`? If I read it like this, it looks like you are rendering two pages at the same time. 

Edit:

I would probably add the `Planner` component to your `HomePage` instead. Right now you render both pages and for both you set similar meta data with helmet.

Reply:
Hi Lene, 
thanks for your review.
The planner page will be a new page. I will implement a navbar after I finished the page. 
The homepage will be filled with other things then.